### ISSUE TITLE
MONDO:0009477-biolink:has_phenotype-MONDO:0009022 - should trigger a "semantic" error based on Biolink Model 

### ISSUE THREAD

**Issue by sierra-moxon** [NCATSTranslator/Feedback#144]
Run what drugs may treat Stromme Syndrome https://ui.ci.transltr.io/results?q=993bc43b-5d43-4f7f-a834-3eb2d6bab21c

review `rosemary extract` path in ARAX:  https://arax.ncats.io/?r=993bc43b-5d43-4f7f-a834-3eb2d6bab21c

<img width="1157" alt="Screen Shot 2023-02-28 at 3 25 14 PM" src="https://user-images.githubusercontent.com/4561292/222006155-a2fcaa82-712c-406a-ab3b-079ccdcb0a85.png">

The edge, "MONDO:0009477----biolink:has_phenotype----MONDO:0009022" can not be found in Monarch Initiative, the primary knowledge source.  BTE is the aggregator knowledge source for this edge.
 
"biokink:has_phenotype" has a range of "phenotypic feature" so immediately this edge makes me wonder if there is a mistake here, or of this conflation is done automatically to increase search coverage?  Is it BTE or NodeNormalizer doing this?  If NN, should it register that object category rewrite with an infores? 

@andrewsu @gaurav

**Comment by andrewsu** [comment]
TLDR: BTE is working as designed. I think the root cause of the confusion is NN's apparent conflation of Diseases and PhenotypicFeatures.  Although that's [a conflation I agree with](https://github.com/biothings/BioThings_Explorer_TRAPI/issues/444#issuecomment-1124189459), so I'd support expanding the range of `"biolink:has_phenotype"` accordingly...

----

My steps for looking into this:
* [Search the translator registry](https://github.com/search?q=repo%3ANCATS-Tangerine%2Ftranslator-api-registry%20infores%3Abiolink-api&type=code) for `infores:biolink-api`:  -> get to the entry for the [biolink API](https://github.com/NCATS-Tangerine/translator-api-registry/blob/master/biolink/openapi.yml)
* Search for "phenotype" for find the section on the endpoint `bioentity/disease/{id}/phenotypes` ([link](https://github.com/NCATS-Tangerine/translator-api-registry/blob/master/biolink/openapi.yml#L169))
* Examine the output for the corresponding API call https://api.monarchinitiative.org/api/bioentity/disease/MONDO:0009477/phenotypes
    * Although I don't find an entry for `MONDO:0009022`, I do find an entry for `HP:0001274` with label "Agenesis of corpus callosum" (see screenshot below)
    * Confirmed that node normalizer says the canonical identifier for `HP:0001274` is `MONDO:0009022` https://nodenorm.transltr.io/1.3/get_normalized_nodes?curie=HP:0001274&conflate=true


![image](https://user-images.githubusercontent.com/2635409/222014197-1576acb9-1c55-425a-b59d-1ba4cf9ac097.png)

**Comment by sierra-moxon** [comment]
I no longer see this returning! :) closing as complete.
